app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
+attributes :content_view_id, :content_view_name
+attributes :lifecycle_environment_id, :lifecycle_environment_name
+attributes :uuid
+
+node :errata_counts do |content_facet|
```

Comment:
i was basically implementing what i thought made sense + what is needed for the current index page. is there other content you think should be there?